libmscore/style.cpp

```diff
@@ -434,7 +434,7 @@ static const StyleType styleTypes[] {
       { Sid::mmRestNumberPos,         "mmRestNumberPos",         Spatium(-1.5) },
       { Sid::hideEmptyStaves,         "hideEmptyStaves",         QVariant(false) },
       { Sid::dontHideStavesInFirstSystem,
-                                 "dontHidStavesInFirstSystm",    QVariant(true) },
+                                 "dontHideStavesInFirstSystem",  QVariant(true) },
```

Comment:
Does this name change cause compatibility issues with existing scores?

Reply:
It may (read: will) cause downrev issues though.

Reply:
Ah, I missed that one.